backend/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Generated by Cargo
+/target/
+
```

Comment:
Many of these paths are already included in the root level .gitignore. Could you move the new paths into the single .gitignore file so we don't have to manage multiple files?